test/unittest/bamboo_test.go

```diff
@@ -71,29 +70,30 @@ var BambooCorrectVariables = map[string]interface{}{
 		},
 		"cluster_security_group": "dummy-sg",
 	},
-	"vpc": VpcDefaultModuleVariable,
-	"pvc_claim_name": "dummy_pvc_claimname",
-	"db_allocated_storage": 5,
-	"db_instance_class":    "dummy_db_instance_class",
-	"db_iops":              1000,
-	"license":              "dummy_license",
-	"admin_username":       "dummy_admin_username",
-	"admin_password":       "dummy_admin_password",
-	"admin_display_name":   "dummy_admin_display_name",
-	"admin_email_address":  "dummy_admin_email_address",
-	"ingress":              map[string]interface{}{},
-	"dataset_url":          nil,
+	"vpc":                     VpcDefaultModuleVariable,
```

Comment:
This re-formmat is already delivered as part of TFKUBE-315. Please pull the latest code to make this PR cleaner.